build/manifest/main.go

```diff
@@ -62,6 +111,30 @@ func findManifest() (*model.Manifest, error) {
 		return nil, errors.Wrap(err, "failed to parse manifest")
 	}
 
+	// Update the manifest based on the state of the current commit
+	// Use the first version we find (to prevent causing errors)
+	var version string
+	tags := strings.Fields(BuildTagCurrent)
+	for _, t := range tags {
+		if strings.HasPrefix(t, "v") {
+			version = t
+			break
+		}
+	}
+	if version == "" {
+		if BuildTagLatest != "" {
+			version = BuildTagLatest + "+" + BuildHashShort
+		} else {
+			version = "v0.0.0+" + BuildHashShort
+		}
+	}
+	manifest.Version = strings.TrimPrefix(version, "v")
```

Comment:
I quite like the version bump PRs as they provide a rendezvous for all parties involved in the release process. That rendezvous point is not so important for plugins maintained by MM, but for Community-maintained ones, where communication happens mostly on GitHub, it is helpful.

With that in mind, I wonder if there is a way we can allow plugin devs to pick if they want dynamic versions or file defined ones, e.g. only use dynamic versions if the `version` field is empty in the file. This way, it's an opt-in feature, which preserves the workflow for Community-maintained plugins.